include/integratorxx/quadratures/radial/lmg.hpp

```diff
@@ -0,0 +1,127 @@
+#include <integratorxx/util/lambert_w.hpp>
+#include <integratorxx/quadratures/radial/radial_transform.hpp>
+
+namespace IntegratorXX {
+namespace lmg {
+
+// Eq 19 of LMG paper (DOI 10.1007/s002140100263)
+inline double r_upper_obj(int m, double alpha, double r) {
+  const double am_term = (m + 1.0) / 2.0;
+  const double g_term  = std::tgamma((m + 3.0) / 2.0);
```

Comment:
Since `m` is an integer, this should be written in terms of a factorial and `sqrt(M_PI)`.

Reply:
Fixed